manage.py

```diff
@@ -1,6 +1,15 @@
 import connexion
 from utils import combine_specifications
+from firebase_admin import credentials, initialize_app
 
+
+# Initializing Firestore database
+# Can import firestore_db to utilize database
+def initialize_firebase():
+    initialize_app(credentials.Certificate("serviceAccountKey.json"))
+
+
+initialize_firebase()

```

Comment:
Wait so why is this enclosed in a function and not just run directly?

Reply:
Functions are generally easier to test. And I might want to import it in a end-to-end test.

Reply:
But for now it would be the same you're right.